doc/tools/docgram/fullGrammar

```diff
@@ -1912,10 +1912,10 @@ hloc: [
 |
 | "in" "|-" "*"
 | "in" ident
-| "in" "(" "Type" "of" ident ")"
-| "in" "(" "Value" "of" ident ")"
 | "in" "(" "type" "of" ident ")"
 | "in" "(" "value" "of" ident ")"
+| "in" "(" "Type" "of" ident ")"      (* SSR plugin *)
+| "in" "(" "Value" "of" ident ")"      (* SSR plugin *)
```

Comment:
Can you look into why this syntax is reintroduced by the SSR plugin?

Reply:
from https://github.com/coq/coq/blob/7e4a34f67ad993fc3742cbb1600b330b3d31a7c6/plugins/ssr/ssrvernac.mlg#L350-L358
no idea what this code is for
cc @coq/ssreflect-maintainers

Reply:
that may be bacuse of is a kwd and hence ssr did repair the orig grammar.
please remove.